labs/a1.md

```diff
@@ -116,5 +116,38 @@ ab/abizer/some/path
 
 Hint: The easiest way to do this is with one very short command.
 
+## Question 4
+
+At some point, everyone has looked at a problem and thought to themselves:
+"Hey, I can do this in one line!"
+
+Lets find out if you can. I need to sort out some of my most listened to albums
+by making directories for each of their songs, specifically for my favorite artist Future.
+
+You need to fetch the list of albums and their respective artist from
+`https://raw.githubusercontent.com/0xcf/decal-labs/master/a1/albums.txt`.
+Filter out the albums we want, trim out the album name,
+and then make a directory for each one, all in one line.
+
+```
+$ cat albums.txt
+...
+Drip or Drown 2, Gunna
+Playboi Carti, Playboi Carti
+DS2 (Deluxe), Future
+Drip Harder, Lil Baby
+...
+
+# After our magic one liner!
+
+$ ls
+'DS2 (Deluxe)'   ...
+```
+
+Hint: Use `man` to see what arguments you need to xargs to behave the way you want.
```

Comment:
I don't think its that crazy to introduce, especially because the beginner lab 1 also uses it (lol). Yeah the biggest problem that @edricx highlighted is how xargs behaves differently on different machines. Perhaps there is a simpler command that could do the same thing that I don't know about? I have to tweak the hints for sure though, I dont want people to think that there is one way to do things.